src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,86 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private File file;
+
+    public Storage(String pathname) {
+        this.file = new File(pathname);
+    }
+
+    public List<Task> loadTasksFromFile() throws DukeException {
+        List<Task> taskList = new ArrayList<>();
```

Comment:
should taskList be written with a plural word instead? ie. tasks

Reply:
Good catch!